### PR TITLE
Merge Transifex 2022-01-08

### PR DESCRIPTION
**Pull request by adipose**

### CODE REVIEW

File: src/mpc-hc/mpcresources/PO/mpc-hc.ca.strings.po [hunk 1]
```diff
@@ -4675,7 +4675,11 @@ msgstr "Dispositiu de so:"
 
 msgctxt "IDS_ARS_RELEASE_DEVICE_IDLE"
 msgid "Release device when idle"
+<<<<<<< HEAD
```
Comment (clsid2) [inline src/mpc-hc/mpcresources/PO/mpc-hc.ca.strings.po]: Merge issue

File: src/mpc-hc/mpcresources/PO/mpc-hc.fr.strings.po [hunk 2]
```diff
@@ -4269,7 +4269,7 @@ msgstr "/dvdpos T#C\tDémarre la lecture du titre T, à partir du chapitre C"
 msgctxt "IDS_CMD_DVDPOS_TIME"
 msgid "/dvdpos T#P\tStart playback at title T, position P (hh:mm:ss)"
 msgstr ""
-"/dvdpos T#P\tDémarrer la lecture du titre T, à partir de la position P "
+"/dvdpos T#P⇥Démarrer la lecture du titre T, à partir de la position P "
```
Comment (clsid2) [inline src/mpc-hc/mpcresources/PO/mpc-hc.fr.strings.po]: This one and one below are wrong.

Comment (adipose) [reply src/mpc-hc/mpcresources/PO/mpc-hc.fr.strings.po]: What is that, a unicode tab or something?

Comment (adipose) [reply src/mpc-hc/mpcresources/PO/mpc-hc.fr.strings.po]: It's U+21e5.  Weird.  It shows a tab in transifex, but that file isn't 100% so it didn't sync yet.  Forcing a sync.

**Comment by adipose** [comment]
I think I know what happened.  The interface of Transifex has changed a bit, and it now shows special glyphs for control characters (such as ⇥ which is RIGHTWARDS ARROW TO BAR' (U+21E5) to represent a tab character).  In order to copy it from the original (english) string, you have to click on the glyph, and it will paste it below for you.  But if you copy/paste your translation from transifex to transifex, it now is pasting that actual unicode char as a literal.

Transifex manual syncing leaves something to be desired.  It's hard to sync a single file.  In this case, there were 7 untranslated strings so I just did them as well as I could with my so-so French, and that caused it to sync due to reaching 100%, There are other flaws, like the fact that I corrected one string, but it didn't sync, because the file was already at 100%.  To make it sync, I had to clear a string, and add it back...

**Comment by clsid2** [comment]
Sounds annoying. Maybe you can ask them to add an option to disable the special character handling.

**Comment by adipose** [comment]
Yes, I left them a list of complaints.  They already responded to me, so I"m hopeful.

**Pull request merged**